frontend/public/components/environment.jsx

```diff
@@ -210,9 +243,24 @@ export const EnvironmentPage = connect(stateToProps)(
             op = 'replace';
           }
         }
-        return {path, op, value: this._envVarsToNameVal(finalPairsForContainer)};
+        return {path, op, value: this._envVarsToNameVal(finalPairsForContainer[EnvType.ENV])};
       });
-      const promise = k8sPatch(model, obj, patch);
+
+      const envFromPatch = currentEnvVars.map((finalPairsForContainer, i) => {
+        let op = 'add';
+        const path = _.isArray(rawEnvData) ? `/${envPath.join('/')}/${i}/envFrom` : `/${envPath.join('/')}/envFrom`;
```

Comment:
Is there a case where envFrom is supported outside of a pod templates, which is always going to be an array?

We might need to support that someday for builds, but I'm wondering if we shouldn't keep things simple in the meantime.

Reply:
Yes, we can support builds by removing this check. I had builds coded, but realized it wasn't supported. We would just need to adjust showing/hiding the envfrom section.